.github/workflows/finish-release.yml

```diff
@@ -0,0 +1,126 @@
+
+name: Finalise a new release
+
+on:
+  pull_request:
+    types:
+    - closed
```

Comment:
Should this not also include some filter on the release branch naming convention? Now this will also trigger on any PR being closed if I understand correctly.

Reply:
oh that's a very good catch! (that's the downside of working in a simplified test environment I guess 😅). Will fix

Reply:
I've fixed this now, though thinking about it, if this ran on a random PR, there wouldn't be any artifacts for it to download so it wouldn't have succeeded anyway. I guess that's one upside to doing it the way I have